### ISSUE TITLE
The graphs don't show on the tensorboard

### ISSUE THREAD

**Issue by resuly** [tflearn/tflearn#701]
I'm a beginner of the TFlearn. I got a question from tensorboard.
**On the tensorboard page, everything is missing expect EMBEDDINGS. I want to get the completed visualization.**
Here is my code (mostly from example):
```
import numpy as np
import tflearn

from tflearn.datasets import titanic
titanic.download_dataset('titanic_dataset.csv')

from tflearn.data_utils import load_csv
data, labels = load_csv('titanic_dataset.csv', target_column=0, columns_to_ignore=[2,7], categorical_labels=True, n_classes=2)

def handleSex(data):
    for person in data:
        person[1] = 1 if person[1] == 'male' else 0
    return np.array(data, dtype=np.float32)
data = handleSex(data)

# init
tflearn.init_graph(num_cores=8, gpu_memory_fraction=0.5)

# Build Neural Network
net = tflearn.input_data(shape=[None, 6])
net = tflearn.fully_connected(net, 32, activation='relu', name="Layer1")
net = tflearn.fully_connected(net, 32, activation='relu', name="Layer2")
# net = tflearn.dropout(net, 0.8, name="Dropout")
net = tflearn.fully_connected(net, 2, activation='softmax', name="Layer3")
net = tflearn.regression(net, optimizer='adam', learning_rate=0.0005, loss='categorical_crossentropy')

model = tflearn.DNN(net, tensorboard_verbose=3, tensorboard_dir="logs",)

# Training & Save weights
model.fit(data, labels, n_epoch=8, batch_size=16, show_metric=True)
model.save('titanic.model')
```
I set the tensorboard_verbose=3, but I can't get the result like [this](http://tflearn.org/getting_started/#visualization)

What else should I set for graphs? Thank you.

**Comment by aymericdamien** [comment]
Everything seems fine. Note that sometimes it takes a long time to appears, so you may wait a few minutes that tensorboard load all data.

**Comment by resuly** [comment]
Thanks for reply. It works now.

**Comment by LyanJin** [comment]
@resuly how did you solve this problem

**Comment by CWOA** [comment]
Same problem here, nothing displays after running @resuly 's code and waiting for 5 minutes..

**Comment by begrs** [comment]
TL;DR
Add "--host=127.0.0.1" to your tensorboard call.

Long Version:
Same here. Nothing except embeddings is shown with tensorboard_verbose=3. The log directory shown in the tensorboard web ui is also the correct one.
I tried waiting for 5min, restarting tensorboard task and reloading the web ui.

What finally solved the issue for me was changing the host of tensorboard to 127.0.0.1 instead of the default localhost! I do not recall were I read this weird work around, but I wanted to share it anyway.

I use this small batch script to execute tensorboard. (The "tensorboard.lnk" located in the same directory is just my way to avoid adding tensorboard to the system environment path variable.)
```
start "Tensorboard" ".\tensorboard.lnk" --logdir=".\tflearn_logs" --host=127.0.0.1
```

**Comment by NishkarshTripathi** [comment]
I have trained a CNN model using tflearn. Now I want to access the logs of my trained model and plot the graph between training VS validation loss and accuracy using matplotlib. Could anyone help me out please?
